script.js

```diff
@@ -1,4 +1,79 @@
+/*
+---REQUIREMENTS---
+[x] Program should be able to take in a text input "scissors", "paper", "stone"
+[x] Program randomly chooses scissors, paper, stone
+[x] User input and program choice can be compared and determined whether user wins, loses or draw
+[x] After comparing user and program choices, frontend page will display the result
+[ ] Program should be able to validate that the input falls within the 3 options
+[ ] Format the output to make it look nicer.
+[ ] 
+*/
+
```

Comment:
Nice way of tracking your work within the code and also helping to explain what it does